keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -501,73 +429,151 @@ version will prevent pods from starting.
 
 ### Rollout, Upgrade and Rollback Planning
 
-Will be added before the transition to beta.
-
 * **How can a rollout fail? Can it impact already running workloads?**
 
+A rollout could fail because the implementation turns out to be
+faulty. Such bugs may cause unexpected shutdowns of kube-scheduler,
+kube-apiserver, kube-controller-manager and kubelet. For the API
+server, broken support for the new volume type may also show up as 5xx
+error codes for any object that embeds a `VolumeSource` (Pod,
+StatefulSet, DaemonSet, etc.).
+
+Already running workloads should not be affected.
+
 * **What specific metrics should inform a rollback?**
 
+One indicator are unexpected restarts of the cluster control plane
```

Comment:
I think these may be too broad. How do we know it's caused by this feature? Can we add metrics in the new controller to track feature usage?

Reply:
I agree these doesn't have to be cause by this feature, but they potentially can.
So understanding/debugging them is a later thing to do after the rollback, but I think we should put them here.

Re metrics - there is a question about that below.